### DISCUSSION TITLE
Portal Berechnung der Tages/Monatsleistung erfolgt nicht mehr und Ereignisreport (es werden immer die gleichen Geräte gefunden)

### DISCUSSION THREAD

**Discussion by Slicery1** [camueller/SmartApplianceEnabler#583] [Installation und Betrieb]
Hallo, 
ich betriebe den SAE seit einiger Zeit. Ich übertrage den aktuellen Verbrauch von mehreren Verbrauchern an den SHM2. Das wird so auch im Portal angezeigt. Das Portal rechnet jedoch nur von meinem größten Verbraucher, nämlich der Wärmepumpe, den Tages/Monatsverbrauch. Mir ist das Problem heute aufgefallen. Das ausbleiben der Berechnung dauert nun bereits seit Anfang Dezember an. Ich schaue jedoch regelmässig in meine App. Und hier habe ich gesehen, dass die Verbräuche der einzelnen Verbraucher Übertrage werden. 

Kennt ihr das Problem? Woran liegt das? 

Weiter wird in unregelmässigen Abständen mittels Ereignisreport mitgeteilt, dass neue Geräte (= die alten Geräte) gefunden wurden. 

Kennt ihr auch das Problem?

Viele Grüße

**Comment by camueller** [comment]
> Kennt ihr das Problem? Woran liegt das?

Du solltetst zuerst prüfen, dass der SAE an der SEMp-Schnittstelle korrekte Daten and den Sunny Home Manager übermittelt.
Wenn das der Falls ist, solltest Du sie auch im Sunny Portal sehen. Wenn nicht, kannst Du Dich an den SMA-Support wenden, wobei die bei "nicht-offiziellen" Produkten wie dem SAE vermutlich nichts machen.

> Weiter wird in unregelmässigen Abständen mittels Ereignisreport mitgeteilt, dass neue Geräte (= die alten Geräte) gefunden wurden. Kennt ihr auch das Problem?

Das ist normal, warum auch immer.

**Reply by Slicery1** [reply]
Danke für deine schnelle Antwort. Zu Punkt 1:

Mir werden die aktuellen Verbrauchswerte im Portal und der app angezeigt. Nur die Berechnung erfolgt nicht.

**Reply by camueller** [reply]
Dann bleibt wohl nur der SMA-Support. Alternativ kannst Du den betroffenen Geräten im SAE neue IDs geben, diese "neuen" Geräte im Sunny Portal hinzufügen (nachdem Du die alten gelöscht hast) und hoffen, dass dann die Berechnung korrekt erfolgt.

**Reply by Slicery1** [reply]
Durch das Löschen verliere ich jedoch alle bisherigen Werte, oder?

**Reply by camueller** [reply]
Ja, aber dann hast Du hoffentlich wenigsten zukünftig saldierte Werte. Oder Du versuchst Dein Glück beim SMA-Support.

**Comment by Slicery1** [comment]
Die Lösung für SMA ist: Geräte sind nicht kompatibel, damit haben wir hiermit nichts zu tun. 
Ticket geschlossen.

Schon sehr schwach von SMA. Die Einzeldaten werden im Portal angezeigt (die Daten liegen somit vor), es werden aber keine Tagesverbräuche ausgerechnet.
